test/integration/testsuite/minishift.go

```diff
@@ -266,11 +266,18 @@ func (m *Minishift) imageShouldHaveCached(image string) error {
 	return util.CompareExpectedWithActualMatchesRegex(image, strings.TrimRight(cmdOut, "\n"))
 }
 
-func (m *Minishift) getOpenShiftUrl() string {
+func (m *Minishift) getOpenShiftConsoleUrl() string {
 	cmdOut, _, _ := m.runner.RunCommand("console --url")
 	return strings.TrimRight(cmdOut, "\n")
 }
 
+func (m *Minishift) getOpenShiftInstanceUrl() string {
+	cmdOut, _, _ := m.runner.RunCommand("ip")
+	ip := strings.TrimRight(cmdOut, "\n")
+	url := "https://" + ip + ":8443"
+	return url
+}
+
```

Comment:
@agajdosi @amitkrout  I have kept the getOpenShiftConsoleUrl() function as I thought you might to use it later. But if you want I can remove it also.